pytrip/cube.py

```diff
@@ -730,6 +730,7 @@ def read(self, path):
 
         :param str path: Path to filename to be read, file extention may be given but is not neccesary.
         """
+        self.basename =  os.path.basename(path).split(".")[0]
```

Comment:
very sad story. In TRiP98 the patientname must be equal to the file name to *some extent*.
using patient_name is not safe, as it may be differen from DICOM import and the like.

Also file name can be suffixed with three digits, and TRiP98 will still accept them as a running number for a plan. So a patient with the patient name TST000 may have the dose cube TST001001 TST001002 TST001003 and the like.

Unfortunately the filename is not store anywhere in the file, so when passing cube objects along to e.g. tripexecuter, this info is lost. Thefore hte basename (i.e. without file extention) must be stored somewhere in the cube object (same for VDX).